lte/gateway/c/session_manager/PipelinedClient.h

```diff
@@ -128,7 +128,7 @@ class PipelinedClient {
       const std::string& imsi, const std::string& ip_addr,
       const std::string& ipv6_addr, const Teids default_teids,
       const std::string& msisdn, const optional<AggregatedMaximumBitrate>& ambr,
-      const RulesToProcess to_process,
+      const RulesToProcess to_process, uint16_t shard_id,
```

Comment:
Same note as above here. ShardID should come before `const RulesToProcess to_process`.